I.pass-tests/06-remove-middle/remove-middle.js

```diff
@@ -0,0 +1,7 @@
+function removeMiddle(array) {
+    if (array.length >= 3) {
+        let index = (array.length - 1) / 2
+        return array.splice(index, 1)
+    } else { return "Can't work with such short array" }
```

Comment:
I like the creativity, but why can't work with short arrays 
also what if it's not odd number ?